### PR TITLE
https://issues.redhat.com/browse/ACM-5462 Expanding pv dynamic storage

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-5462

### CODE REVIEW

File: observability/observability_enable.adoc [hunk 1]
```diff
@@ -378,6 +378,8 @@ The Thanos version is displayed.
 [#creating-mco-cr]
 === Creating the MultiClusterObservability custom resource
 
+Use the `MultiClusterObservability` custom resource to specify the persistent volume storage size for various components. You must set the storage size during the initial creation of the `MultiClusterObservability` custom resource. When you update the storage size values post-deployment, changes take effect only if the storage class supports dynamic volume expansion. For more information, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.5/html/storage/expanding-persistent-volumes[Expanding persistent volumes from the {ocp} documentation].
```
Comment (dockerymick) [inline observability/observability_enable.adoc]:
```suggestion
Use the `MultiClusterObservability` custom resource to specify the persistent volume storage size for various components. You must set the storage size during the initial creation of the `MultiClusterObservability` custom resource. When you update the storage size values post-deployment, changes take effect only if the storage class supports dynamic volume expansion. For more information, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/storage/expanding-persistent-volumes[Expanding persistent volumes from the {ocp} documentation].
```

**Review by subbarao-meduri — APPROVED** [review]
/lgtm

File: observability/observability_enable.adoc [hunk 2]
```diff
@@ -378,6 +378,8 @@ The Thanos version is displayed.
 [#creating-mco-cr]
 === Creating the MultiClusterObservability custom resource
 
+Use the `MultiClusterObservability` custom resource to specify the persistent volume storage size for various components. You must set the storage size during the initial creation of the `MultiClusterObservability` custom resource. When you update the storage size values post-deployment, changes take effect only if the storage class supports dynamic volume expansion. For more information, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/storage/expanding-persistent-volumes[Expanding persistent volumes from the {ocp} documentation].
```
Comment (cadawson) [inline observability/observability_enable.adoc]:
```suggestion
Use the `MultiClusterObservability` custom resource to specify the persistent volume storage size for various components. You must set the storage size during the initial creation of the `MultiClusterObservability` custom resource. When you update the storage size values post-deployment, changes take effect only if the storage class supports dynamic volume expansion. For more information, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/storage/expanding-persistent-volumes[Expanding persistent volumes] from the {ocp} documentation.
```

Comment (cadawson) [reply observability/observability_enable.adoc]: I typically include only the name of the topic in the link.

**Review by cadawson — APPROVED** [review]
One minor suggestion. Looks good!

**Comment by dockerymick** [comment]
@subbarao-meduri @cadawson thank you both for your review and approval. Merging this issue

**Pull request merged**